Makefile

```diff
@@ -203,6 +203,13 @@ test-verbose: build-tests
 test-verbose:
 	PYTHONFAULTHANDLER=1 python -m pytest $(COV_FLAGS) --ignore ./tests/docker -vv -s ./tests
 
+# help: test-verbose-with-coverage                   - Build and run all tests [verbose-with-coverage]
+.PHONY: test-verbose-with-coverage
+test-verbose-with-coverage: build-tests
+test-verbose-with-coverage: CMAKE_ARGS="-DCOVERAGE=on"
+test-verbose-with-coverage:
```

Comment:
Do you need a backslash on this line to get the next executed as a dependency?  Also, the first two dependencies could have been put together on one line

Reply:
The current version is within the expected way of writing a Makefile. In general though, we really do need to rework this Makefile.